test/functional/api/cas/cache_config.py

```diff
@@ -158,6 +158,14 @@ def __eq__(self, other):
             and self.wake_up_time == other.wake_up_time
         )
 
+    def __str__(self):
+        ret = ""
+        if self.flush_max_buffers is not None:
+            ret += f"acp flush max buffers value {self.flush_max_buffers} "
+        if self.wake_up_time is not None:
+            ret += f"acp wake up time value {self.wake_up_time}"
+        return ret
```

Comment:
If somebody just change cleaning policy to ACP without setting any param it would return an empty string.
Maybe in case like that, this method should return values taken from `get_flush_parameters_acp()`?